thread/src/main/kotlin/io/github/paulgriffith/kindling/thread/ThreadComparisonPane.kt

```diff
@@ -367,9 +432,11 @@ class ThreadComparisonPane(
         private val blockedIcon = FlatSVGIcon("icons/bx-block.svg").derive(12, 12)
         private val detailIcon = FlatSVGIcon("icons/bx-link-external.svg").derive(12, 12)
 
-        private val percent = NumberFormat.getPercentInstance()
+        private val percent = DecimalFormat("0.000'%'")
 
-        private const val SHOW_NULL_THREADS_DEFAULT = true
+        private const val SHOW_NULL_THREADS_DEFAULT = false
         private const val SHOW_EMPTY_VALUES_DEFAULT = false
+
+        private val THREAD_HIGHLIGHT_COLOR  = UIManager.getColor("Component.warning.focusedBorderColor")
```

Comment:
Is this color the same on different themes?

Reply:
@paul-griffith It's slightly different but I don't think it's noticeable.. It's a color that I and a few others agreed had a good amount of contrast on both themes.

Reply:
How about we just make this a computed getter? Should be pretty fast and won't get called _that_ often.